test/tools/cli/IonProcess.java

```diff
@@ -0,0 +1,312 @@
+package tools.cli;
+
+import com.amazon.ion.*;
+import com.amazon.ion.system.IonBinaryWriterBuilder;
+import com.amazon.ion.system.IonReaderBuilder;
+import com.amazon.ion.system.IonTextWriterBuilder;
+import org.kohsuke.args4j.Argument;
+import org.kohsuke.args4j.CmdLineException;
+import org.kohsuke.args4j.CmdLineParser;
+import org.kohsuke.args4j.Option;
+import tools.events.Event;
+import tools.events.EventType;
+import tools.events.ImportDescriptor;
+
+
+import java.io.*;
```

Comment:
Let's list every import individually, not using `*`.

Reply:
Changed, My IDE did this automatically once I have 5+ imports from the same package.